src/ontology/uberon-edit.obo

```diff
@@ -224223,6 +224223,14 @@ intersection_of: part_of UBERON:0007318 ! saphenous vein
 relationship: dc-contributor https://orcid.org/0000-0002-0819-0473 ! Paula Duek Roggli
 property_value: dcterms-date "2023-08-31T11:41:09Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480069
+name: temporofacial region
+def: "The region at each side of the face comprising the temple and the cheek." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0001444 ! subdivision of head
```

Comment:
- Given the textual definition, consider adding `'part of' some face`.
- If cheek is part of this region, consider adding the axiom in cheek `'part of' some 'temporofacial region'`.

Reply:
@aleixpuigb what is better: `'part of' some face` or `'part of' some temple`?

Reply:
I don't think it is `'part of' some temple`, according to the definition:
> The region where the frontal, parietal, temporal, and sphenoid bones fuse.

After a cursory research I would add the following three axioms:
- `overlaps some face` (the temple is not considered part of the face).
- The temple is `'part of' some 'temporofacial region'`
- The cheeck is `'part of' some 'temporofacial region'`